modules/installation-ibm-cloud-config-yaml.adoc

```diff
@@ -102,10 +107,107 @@ endif::openshift-origin[]
 ====
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
 ====
+endif::with-networking,without-networking[]
+
+ifdef::vpc[]
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+controlPlane: <2> <3>
+  hyperthreading: Enabled <4>
+  name: master
+  platform:
+    ibm-cloud: {}
+  replicas: 3
+compute: <2> <3>
+- hyperthreading: Enabled <4>
+  name: worker
+  platform:
+    ibmcloud: {}
+  replicas: 3
+metadata:
+  name: test-cluster <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14 <5>
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+ifndef::openshift-origin[]
+  networkType: OpenShiftSDN
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+  networkType: OVNKubernetes
+endif::openshift-origin[]
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  ibmcloud:
+    region: eu-gb <1>
+    vpcName: eu-gb-example-network-1 <6>
+    controlPlaneSubnets: <7>
```

Comment:
resourceGroupName: eu-gb-example-network-rg
the resource group of the vpc

Reply:
I did add `resourceGroupName` to the sample file, but did not explain the parameter again. The parameter is described in  [Additional IBM Cloud VPC configuration parameters](https://50357--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-vpc.html#installation-configuration-parameters-additional-ibm-cloud_installing-ibm-cloud-vpc)